components/webui/client/src/pages/SearchPage/SearchControls/Presto/BuildSqlTestingInputs.tsx

```
@@ -0,0 +1,47 @@
+    return (
+        <form
+            onSubmit={(ev: FormEvent<HTMLFormElement>) => {
+                ev.preventDefault();
+                const formData = new FormData(ev.target as HTMLFormElement);
+                const props = Object.fromEntries(formData) as unknown as BuildSearchQueryProps;
+                const sqlString = buildSearchQuery(props);
+                console.log(`SQL: ${sqlString}`);
+                handlePrestoQuerySubmit({queryString: sqlString});
+            }}
+        >
+            <label>select:</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)